napari/layers/labels/_labels_key_bindings.py

```diff
@@ -1,6 +1,9 @@
+from typing import cast
+
 import numpy as np
 from app_model.types import KeyCode, KeyMod
 
+from napari.components.overlays.labels_polygon import LabelsPolygonOverlay
```

Comment:
This seem to be the line that cause errors. 
This is typically why I think relative import are better, as you can make sure that you don't import from cousins' submodules, and have a tree and not a DAG. But it seen napari prefers absolute imports.

I think we can use `if TYPE_CHECKING:` and use a str in the `cast`

Reply:
I pushed the commit to fix the circular import.